### PR TITLE
feat: improved ladder section

### PR DESCRIPTION
**Pull request by urana123**

### CODE REVIEW

File: frontend/src/components/profile/VerticalTabs.tsx [hunk 1]
```diff
@@ -1,14 +1,14 @@
 import { FC, ReactNode, SyntheticEvent, useState } from "react";
 import { Tabs, Tab, Box, Stack } from "@mui/material";
-import Level from "./Level";
+import Ladder from "./ladder";
```
Comment (GereltuyaMz) [inline frontend/src/components/profile/VerticalTabs.tsx:3]: It should named "level"

File: frontend/src/components/profile/ladder/CommentList.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    borderRadius: 4,
+    padding: 3,
+    maxWidth: "500px",
+  },
+  typo: {
+    fontSize: 15,
+  },
+};
+
+const Alim: FC<LadderCommentItemType> = ({ name, img, comment }) => {
```
Comment (GereltuyaMz) [inline frontend/src/components/profile/ladder/CommentList.tsx:18]: think of another name, pls

File: frontend/src/components/profile/ladder/index.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```
Comment (GereltuyaMz) [inline frontend/src/components/profile/ladder/index.tsx:1]: should not disable eslint, only if this must necessary

File: frontend/src/components/profile/ladder/index.tsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    paddingBottom: 2,
+    marginTop: 2,
+  },
+  divider: {
+    border: "1px solid #f0f2f5",
+  },
+};
+
+const Level: FC = () => {
+  // eslint-disable-next-line no-unused-vars
```
Comment (GereltuyaMz) [inline frontend/src/components/profile/ladder/index.tsx:20]: should not disable eslint

File: frontend/package-lock.json [hunk 5]
```diff
@@ -6641,4 +6641,4 @@
       }
     }
   }
-}
```
Comment (billduluu) [inline frontend/package-lock.json:6644]: ?? ene

**Pull request closed without merging**